vulcan/_message.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+
+from related import (
+    IntegerField,
+    StringField,
+    DateField,
+    ChildField,
+    immutable,
+    to_model,
+)
+
+from ._subject import Subject
+from ._teacher import Teacher
+from ._utils import sort_and_filter_date
+@immutable
+class Message:
+	sender = StringField(key="Nadawca")
+	senderid = StringField(key="NadawcaId")
+	addressees = StringField(key="Adresaci")
+	title = StringField(key="Tytul")
+	content = StringField(key="Tresc")
+	postdate = StringField(key="DataWyslania")
+	posthour = StringField(key="GodzinaWyslania")
+	folder= StringField(key="FolderWiadomosci")
+	"""
+	messageid = ChildField(MessageId, required=False)
```

Comment:
Uncomment this, rename to `id` and make it `IntegerField` with key `"WiadomoscId"`. Move it to the top.